packages/router/router.ts

```diff
@@ -782,7 +778,7 @@ export function createRouter(init: RouterInit): Router {
         }
 
         warning(
-          activeBlocker != null && delta === null,
+          blockerFunctions.size === 0 || delta != null,
```

Comment:
This was the incorrect boolean condition causing the warning, it should have been `activeBlocker == null || delta != null` since we only want to warn if we have a blocker _and_ delta is `null`.